external/gitlab/gitlab.go

```diff
@@ -112,18 +122,34 @@ func POST(instanceURL string, resourcePath string, token string, body io.Reader)
 	}
 
 	req.Header.Set("Content-Type", "application/json")
-	req.Header.Add("Authorization", "Bearer "+token)
+	req.Header.Add("Authorization", "Bearer "+*token)
```

Comment:
I am thinking of writing it a bit differently to avoid duplication, something like

retrier( func (token) {
 url := fmt.Sprintf("%s/%s/%s", instanceURL, APIPath, resourcePath)
....

}

this could be a reference for future contributors as well. wdyt?